src/repositories/participant.js

```diff
@@ -0,0 +1,17 @@
+class ParticipantRepository {
+    constructor(model) {
+        this.model = model;
+    }
+
+    async list() {
+        return this.model.find();
+         
+    }
+
+    async findUser(userName) {
+        let user = {userName : userName};
+        return this.model.find(user);
```

Comment:
Não tem necessidade de declarar uma variável só pra usar ela como filtro, pode fazer direto
```suggestion
        return this.model.find({userName : userName});
```